2-Draft-Accepted/RFCNNNN-Support-Experimental-Features.md

```diff
@@ -457,6 +452,35 @@ as well as a problem that we have to solve before using the experimental feature
 
 > Not sure how to handle it yet ...
 
+### Experimental Feature Incompatibility Mitigation
+
+There will be cases where experimental features are incompatible either intentionally or unintentionally.
+
+- An example of an intentional incompatibility occurs when two alternative, but incompatible solutions, are being tested.
+- An example of an unintended incompatibility can occur when two experimental features prove to be incompatible through usage.
+
+A minimal implementation will support defining a list of experimental feature pairs in the `powershell.config.json` file.
+At startup, attempts to use incompatible features would produce warnings and consider both as disabled.
```

Comment:
We need to be more explicit on how conflicting features would be described, both in `ExperimentalFeature` and in `PrivateData.ExperimentalFeatures` from a module manifest.

Reply:
Dan and I discussed this and decided to not get into details on this section.